crates/solidity/testing/snapshots/cst_output/ContractDefinition/function_multiple_delimiters/generated/0.4.11-failure.yml

```diff
@@ -90,7 +90,7 @@ Tree:
                       - Statement (Rule): # 201..262 "\n    (bool success, ) = recipient.call{ value: amo..."
                           - TupleDeconstructionStatement (Rule): # 201..262 "\n    (bool success, ) = recipient.call{ value: amo..."
                               - OpenParen (Token): "(" # 206..207
-                              - TupleMembersList (Rule): # 207..221 "bool success, "
+                              - TupleMembersList (Rule): # 207..220 "bool success,"
```

Comment:
That's a result of https://github.com/NomicFoundation/slang/pull/620/files#diff-98dac28611f82beed4ba09bf8743d5ab282508ada2c34d88cf39d36617f8fb55L24 - I simplified it so that we don't manually parse whitespace before (and include it in the result) before peeking the significant token/trying the recovery, it's more correct in the sense that this shouldn't be a trailing trivia in this case.

In this case, I believe that `CloseParen` should have the trailing trivia included, but we don't print it in the CST snapshot IIRC, I will double-check that.

In general, I think that we should think whether we should include that in the "correct parse" during error recovery, since there may be cases where it goes against the "tokens only have leading trivia, except possibly the final significant token before the newline" invariant. I can verify and tackle this as part of this PR or in a follow-up, let me know what you'd prefer.